.travis.yml

```diff
@@ -1,12 +1,10 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
   - 5.6
-  - 7.0
-  - hhvm
```

Comment:
Why these being removed? We should continue to test against them.

Reply:
We should resolve any dependency issues rather than entirely drop support for older versions of PHP.